recipes/lfpykit/LICENSE

```diff
@@ -0,0 +1,674 @@
+                    GNU GENERAL PUBLIC LICENSE
```

Comment:
You can delete this file and let conda-build use the one from source archive

Reply:
alright, removed it.